lib/report_plugins/glucosedistribution.js

```diff
@@ -137,10 +135,10 @@ glucosedistribution.report = function report_glucosedistribution(datastorage,day
   if (data.length > 0) {
     var localBgs = data.map(function(r) { return r.sgv; }).filter(function(bg) { return !!bg; });
     var mgDlBgs = data.map(function(r) { return r.bgValue; }).filter(function(bg) { return !!bg; });
-    $('<td class="tdborder">' + Math.round(10*ss.mean(localBgs))/10 + '</td>').appendTo(tr);
-    $('<td class="tdborder">' + Math.round(10*ss.quantile(localBgs, 0.5))/10+ '</td>').appendTo(tr);
-    $('<td class="tdborder">' + Math.round(ss.standard_deviation(localBgs)*10)/10 + '</td>').appendTo(tr);
-    $('<td class="tdborder"><center>' + Math.round(10*(ss.mean(mgDlBgs)+46.7)/28.7)/10 + '%<sub>DCCT</sub> | ' +Math.round(((ss.mean(mgDlBgs)+46.7)/28.7 - 2.15)*10.929) + '<sub>IFCC</sub></center></td>').appendTo(tr);
+    $('<td class="tdborder">' + (Math.round(10*ss.mean(localBgs))/10).toFixed(1) + '</td>').appendTo(tr);
+    $('<td class="tdborder">' + (Math.round(10*ss.quantile(localBgs, 0.5))/10).toFixed(1) + '</td>').appendTo(tr);
+    $('<td class="tdborder">' + ()Math.round(ss.standard_deviation(localBgs)*10)/10).toFixed(1) + '</td>').appendTo(tr);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/968201834/issues/source?bid=2452379&fileBranchId=2860611#l140)